### PR TITLE
Restore AS's filter option on "All Games" page

### PR DESCRIPTION
**Pull request by candela97**
If getting images to load is as simple as https://github.com/tfedor/AugmentedSteam/commit/01e9d69bab059fdfe92eab48ff8cd64c2fecb029 , I think we can consider hiding Steam's option instead, since as mentioned in https://github.com/tfedor/AugmentedSteam/issues/681#issuecomment-564059216 our option does have the advantage of not refreshing the page.

Additionally, https://github.com/tfedor/AugmentedSteam/commit/01e9d69bab059fdfe92eab48ff8cd64c2fecb029 should be applied when hiding uncommon games instead, because that's when games further down with unloaded images potentially get shown.

### CODE REVIEW

File: css/enhancedsteam.css [hunk 1]
```diff
@@ -2897,7 +2897,13 @@ input[type=checkbox].es_dlc_selection:checked + label {
   overflow: hidden;
 }
 
-.esi-hide-common .esi-common {
+/* hide Steam's option on "All Games" page */
+input[id='show_common_games'],
```
Comment (Revadike) [inline css/enhancedsteam.css]: Why single quote?

**Comment by Ryzhehvost** [comment]
We can also make this configurable, so that if user trusts/likes default checkbox more, they can stick to it.

**Comment by MxtOUT** [comment]
> Additionally, 01e9d69 should be applied when hiding uncommon games instead, because that's when games further down with unloaded images potentially get shown.

I don't know why you wouldn't want it to load there too?

Also, just dropping the idea of turning the checkboxes into radio buttons, since you're hiding every game if you have both boxes checked

**Comment by candela97** [comment]
> I don't know why you wouldn't want it to load there too?

Yeah I'll put it back
 
> Also, just dropping the idea of turning the checkboxes into radio buttons, since you're hiding every game if you have both boxes checked

But then we'll need another option to show all games, right?

**Comment by MxtOUT** [comment]
> But then we'll need another option to show all games, right?

Yeah, I forgot that, right

**Pull request merged**